docs/docs/cmd/teams/channel/channel-member-list.md

```diff
@@ -0,0 +1,54 @@
+# teams channel member list
+
+Lists members of the specified Microsoft Teams team channel
+
+## Usage
+
+```sh
+m365 teams channel member list [options]
+```
+
+## Alias
+
+```sh
+m365 teams conversationmember list [options]
+```
+
+## Options
+
+`--teamId [teamId]`
+: The Id of the Microsoft Teams team. Specify either `teamId` or `teamName` but not both
+
+`--teamName [teamName]`
+: The display name of the Microsoft Teams team. Specify either `teamId` or `teamName` but not both
+
+`--channelId [channelId]`
```

Comment:
we should add the -c option as in the conversationmember-list command to avoid breaking changes